subjects/devops/play-with-containers/audit/README.md

```diff
@@ -2,30 +2,29 @@
 
 ##### Check the Repo content
 
-A `README.md` file and all files used to create and delete and manage the student infrastructure must be submitted in the repo.
+A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.
 
 ###### Are all the required files present?
 
-###### `.env` file is not exists in the repo?
+###### `.env` file do not exist in the repo?

```

Comment:
It would make more sense: Does the `.env` file exist in the repo?
Or at least does instead of do.

Reply:
I've fixed `do` to `does`, but we still need `yes` to be the happy path, and the file doesn't need to exist for the audit to pass

Reply:
My bad, I didn't realise that it was in negative. In that case the equivalent would  be: Does the `env` file not exist in the repo?
Or you can leave it as it is and resolve this conversation if you think it's fine.